src/core/types.ts

```diff
@@ -42,7 +42,3 @@ export type DataReferencingMessage = {
 
   encodedData: string;
 };
-
-export type AuthCreateOptions = {
```

Comment:
Intentional? Might break users of the sdk

Reply:
Yes, intentional. This is not used by users, though even if it is, we break users consistently currently right now while we undergo rapid changes and development. Better to break now than later when there are "actually" users by the thousands.